src/cow-react/modules/limitOrders/containers/OrdersWidget/hooks/useLimitOrdersList.ts

```diff
@@ -15,11 +34,31 @@ export function useLimitOrdersList(): LimitOrdersList {
   const allNonEmptyOrders = useOrders({ chainId })
   const accountLowerCase = account?.toLowerCase()
 
+  const ordersFilter = useCallback((order: Order) => order.owner.toLowerCase() === accountLowerCase, [accountLowerCase])
+  const ordersSorter = (a: Order, b: Order) => Date.parse(b.creationTime) - Date.parse(a.creationTime)
+  const ordersParser = (order: Order): ParsedOrder => {
```

Comment:
I have some questions about this function.

- 1. Is it just to get some augmented order info? 
- 2. If so, why `Parsed`? that makes me very confused. Parsing would be for example to converting from saw raw string into a data object for example. If it's just one type with additional data for the receipt, you could just call it `OrderReceipt`? 
- 3. Are we sure this props shouldn't just be created in the presentation layer instead of in this hook?
- 4. This function looks static, why did we define it inside the hook?

Reply:
Hmm this is a tricky one, maybe we can refactor this better